artifact-management/commands/gem-test.yml

```diff
@@ -1,36 +1,16 @@
-description: Perform the actual gem build
```

Comment:
You've made quite a few changes in this file, which are unrelated. They should be in separate commits to tell a cleaner story. Remember that commits tell a story to the developer who reads them later. Make the story as easy as possible to understand.

The change here at line 1 is "correct the description of the `gem-test` command". It's unrelated to the rest of the changes.